{"id": "121687", "title": "Bug 121687: Exception throws when exporting data in a table inside a grid cell.", "description": "", "OB": "When exporting data in a table that is inside a grid cell, an exception is thrown. The exception occurs in the 'ReportEngineService.getMetaData()' method.", "EB": "When exporting data in two tables, the expected result is that the data is exported without any exceptions.", "SR": "To reproduce the issue: 1. Insert a grid. 2. Export data in two tables."}